apps/dev/pages/api/auth/[...nextauth].ts

```diff
@@ -3,6 +3,7 @@ import type { NextAuthOptions } from "next-auth"
 
 // Providers
 import Apple from "next-auth/providers/apple"
+import AsgardeoProvider from "next-auth/providers/asgardeo"
```

Comment:
Can't we rename this to Asgardeo?

Reply:
Then it will be consistent with the display name everywhere

Reply:
This is fixed with the latest commit